srml/balances/src/lib.rs

```diff
@@ -165,6 +159,28 @@ decl_storage! {
 		/// The fee required to create an account. At least as big as ReclaimRebate.
 		pub CreationFee get(creation_fee) config(): T::Balance;
 
+		/// Information regarding the vesting of a given account.
+		pub Vesting get(vesting) build(|config: &GenesisConfig<T>| {
+			config.vesting.iter().filter_map(|&(ref who, begin, length)| {
+				let begin: u64 = begin.as_();
+				let length: u64 = length.as_();
+				let begin: T::Balance = As::sa(begin);
+				let length: T::Balance = As::sa(length);
+
+				config.balances.iter()
+					.find(|&&(ref w, _)| w == who)
+					.map(|&(_, balance)| {
+						// <= begin it should be >= balance
+						// >= begin+length it should be <= 0
+
+						let per_block = balance / length;
+						let offset = begin * per_block + balance;
```

Comment:
Rounding above could have a big effect on `offset` if `begin` is large?

Reply:
I guess rounding can only cause a difference of 1 unit, so at most 1 * `begin`

Reply:
begin will typically be zero. but even when it's not, it'll still be orders of magnitude below cents.